com.abagames.rrotage.json

```diff
@@ -0,0 +1,164 @@
+{
+  "app-id": "com.abagames.rRootage",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "rrootage",
+  "rename-desktop-file": "rrootage.desktop",
+  "rename-icon": "rrootage",
+  "rename-appdata-file": "rrootage.appdata.xml",
+  "finish-args": [
+    "--device=dri",
+    "--socket=pulseaudio",
+    "--socket=wayland",
+    "--socket=x11"
```

Comment:
add --share=ipc if you add socket=x11 http://docs.flatpak.org/en/latest/sandbox-permissions.html